README.md

```diff
@@ -97,6 +97,7 @@ r := render.New(render.Options{
     IsDevelopment: true, // Render will now recompile the templates on every HTML response.
     UnEscapeHTML: true, // Replace ensure '&<>' are output correctly (JSON only).
     StreamingJSON: true, // Streams the JSON response via json.Encoder.
+    DisableHTTPErrorRendering: true, // Don't render http.StatusInternalServerError when errors occur so I can handle errors myself
```

Comment:
How about `Disables automatic rendering of http.StatusInternalServerError when an error occurs.`